platform/src/common/components/TopBar/index.jsx

```diff
@@ -1,51 +1,24 @@
-import AirqoLogo from '@/icons/airqo_logo.svg';
-import HelpCenterIcon from '@/icons/help_center.svg';
-import NotificationsIcon from '@/icons/notifications.svg';
-import ArrowDropDownIcon from '@/icons/arrow_drop_down';
-import SearchIcon from '@/icons/Actions/search.svg';
 import { useState } from 'react';
+import ProfileIcon from '@/icons/SideBar/profile.svg';
+import SearchMdIcon from '@/icons/Actions/search_md.svg';
+import NotificationsIcon from '@/icons/Actions/Notification.svg';
+import TopBarItem from './TopBarItem';
 
 const TopBar = () => {
-  const [searchInput, setSearchInput] = useState();
   return (
-    <nav className='fixed top-0 w-full z-10 px-4 py-3 h-16 box-border border-b-[0.5px] border-b-grey-750 bg-white'>
-      <div className='flex justify-end md:justify-between items-center'>
-        <AirqoLogo className='invisible lg:visible w-[46.56px] h-8' />
-
-        <form className='hidden relative md:block md:max-w-[280px] lg:max-w-[448px] h-10 w-full'>
-          <div className='absolute my-[10px] mx-3'>
-            <SearchIcon />
-          </div>
-          <input
-            type='search'
-            id='search'
-            placeholder='Search'
-            value={searchInput}
-            onChange={(e) => setSearchInput(e.target.value)}
-            className='h-10 w-full bg-grey-250 flex justify-center pl-10 rounded-md text-sm border-0 focus:outline-none focus:ring focus:ring-violet-300'
-          />
-        </form>
-
+    <div className="top-0 w-full bg-white  flex border-b-[1px] sm:px-4 py-3 px-4 lg:px-6 justify-between items-center shadow-sm">
+      <div className="font-bold invisible lg:visible ">Home</div>
+      <div className='invisible lg:visible sm:flex  justify-end md:justify-between items-center '>
         <div className='flex w-auto'>
-          <div className='border border-grey-750 w-10 h-10 rounded-lg ml-3 p-[10px]'>
```

Comment:
![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/9f467350-1ecd-47fb-83c4-25e7cd8d9b31)

Is this topbar supposed to be sticky? @danielmarv Please find out from Paul

Reply:
resolved